class Error(BaseException):
	pass

class ConnectionError(Error):
	pass